packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Okta Hosted Vs. Customer Hosted
+---
+
+## Okta Hosted Vs. Customer Hosted
+
+Okta offers multiple integration options when designing an authentication experience. The two most recommended concepts are using the Okta hosted widget and the customer hosted widget. Both of these Okta built experiences provide easy implementation and maintenance, industry best practice, user self service, and built in user registration. The difference between the two concepts is where the widget itself is hosted.
+
+## Okta Hosted Widget
+
+The Okta hosted widget is considered the easiest and most secure means of integration. This is because the widget itself is hosted by Okta, maintained by Okta, and kept secure by Okta. The Okta hosted widget is recommended for most integrations. OpenID Connect Redirect Flow is used to redirect users to Okta, where the user authenticates and is redirected back to a specified, whitelisted, redirect URI. This design is considered security best practice. The look and feel, as well as the enabled features, are all configured within the Okta Admin UI. HTML customization of the page is accessible in the Admin UI which provides a moderate level of customization to the experience itself.
+
+## Customer Hosted Widget
+
+The customer hosted widget is considered the best balance of flexibility and effort to integrate. The customer hosted widget is recommended if an integration requires a deeper level of customization or a Okta hosted experience is not possible. The customer hosted widget works by providing an open source [JavaScript widget](https://github.com/okta/okta-signin-widget) to embed into the page. The widget is built and updated by Okta, uses industry best practice security design, and is added to a page with a few lines of HTML/JavaScript. The widget itself can be loaded directly from the CDN, NPM, or built from source. The look and feel is customized directly via HTML/CSS/SASS and JavaScript. Features are configured within the Okta Admin UI and enabled in the widget via JavaScript.
```

Comment:
```suggestion
The Customer-hosted widget is considered the best balance of flexibility and effort to integrate. The Customer-hosted widget is recommended if an integration requires a deeper level of customization or a Okta-hosted experience isn't possible. The Customer-hosted widget works by providing an open source [JavaScript widget](https://github.com/okta/okta-signin-widget) to embed into the page. The widget is built and updated by Okta, uses industry best practice security design, and is added to a page with a few lines of HTML/JavaScript. The widget itself can be loaded directly from the CDN, NPM, or built from source. The look and feel is customized directly through HTML/CSS/SASS and JavaScript. Features are configured within the Okta Admin Console and enabled in the widget through JavaScript.
```
Just an FYI link to the more extensive Style Guide than what is here in Git (and where some of these edits are coming from):
https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/722929640/InfoDev+Style+Guide